perspective_reddit_bot/log_subreddit_comments.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tool to log comments to a subreddit."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import argparse
+from datetime import datetime
+import json
+import os
+import sys
+
+import praw
+
+from creds import creds
+
+
+def datetime_timestamp(dt):
+  """Returns compact, filename-friendly string representation of datetime."""
+  return dt.strftime('%Y%m%d_%H%M%S')
+
+
+def now_timestamp():
```

Comment:
Generally I prefer to store timestamps as seconds or milliseconds since the epoch.  The way you are storing them now is more readable but has 2 issues I can think of:
* It is in the local timezone, which might be confusing depending on where the code is run or where the logs are viewed from.
* It will be ambiguous during the daylight savings "fall back", when a time like 1:30am actually happens twice.

Maybe this is OK if these timestamps are just for humans to read, but if you have software reading them it could result in little bugs.

Reply:
Ah, great catch! Changed to use utcnow()